diff --git a/lib/resolver.ts b/lib/resolver.ts
index 3845a6542c924397a7c2e2daf0a78cdb243ec603..0a02155ecf9b7f208432eb7e7c0a1108de502786 100644
--- a/lib/resolver.ts
+++ b/lib/resolver.ts
@@ -1,10 +1,10 @@
-const baseURL = "https://api.astroicon.dev/v1/";
+const baseURL = "https://api.iconify.design/";
 const requests = new Map();
 const fetchCache = new Map();
 
 // Default resolver fetches icons from `api.astroicon.dev`
 export default async function get(pack: string, name: string) {
-  const url = new URL(`./${pack}/${name}`, baseURL).toString();
+  const url = new URL(`./${pack}/${name}.svg`, baseURL).toString();
   // Handle in-flight requests
   if (requests.has(url)) {
     return await requests.get(url);